integrations/gitlab/gitlab_integration/events/event_handler.py

```diff
@@ -75,8 +81,14 @@ async def _notify(self, event_id: str, body: dict[str, Any]) -> None:
                 f"event: {event_id} has no matching handler. the handlers available are for events: {self._observers.keys()}"
             )
             return
-
-        await asyncio.gather(*(observer(event_id, body) for observer in observers_list))
+        for observer in observers_list:
+            if asyncio.iscoroutinefunction(observer):
+                logger.debug(
+                    f"Notifying observer: {observer.__name__}, event: {event_id}"
+                )
+                asyncio.create_task(observer(event_id, body))  # type: ignore
+            else:
+                logger.error(f"Observer {observer} is not a coroutine function")
```

Comment:
if we reach here doesn't it means that something in our code is not right? or am i missing something?
if im right then not sure why we need it

Reply:
you right, will remove